scripts/pull-kinto-admin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+VERSION=$(cat kinto/plugins/admin/VERSION)
+TAG="v${VERSION}"
+
+# download and unzip release
+curl -OL "https://github.com/Kinto/kinto-admin/releases/download/${TAG}/kinto-admin-release.tar"
+rm -r ./kinto/plugins/admin/build || echo "admin/build folder doesn't exist yet"
+mkdir ./kinto/plugins/admin/build
+tar -xf kinto-admin-release.tar -C ./kinto/plugins/admin/build && rm kinto-admin-release.tar
+echo "$VERSION" > ./kinto/plugins/admin/build/VERSION
```

Comment:
Maybe write a comment here that after https://github.com/Kinto/kinto-admin/commit/8400176ae38e4975e43a1dccf7926639c601ba78 is deployed, we don't have to do this step?